daprdocs/content/en/developing-applications/ides/visual-studio-code/vscode-remote-dev-containers.md

```diff
@@ -0,0 +1,31 @@
+---
+type: docs
+title: "Visual Studio Code remote dev containers"
+linkTitle: "Remote dev containers"
+weight: 20000
+description:  "How to setup a remote dev container environment with Dapr"
```

Comment:
I am not a fan of this title since it does not tell you what you can do. Can we call this something like "Developing Dapr applications with remote dev containers". I have always found it confusing just saying remote dev containers, since unless you know what they are, you do not know what they are useful for.

Reply:
Yeah you're totally right!